docs/hdfs_connector.rst

```diff
@@ -54,16 +82,37 @@ The three records entered are published to the Kafka topic ``test_hdfs`` in Avro
 
 Before starting the connector, please make sure that the configurations in
 ``etc/kafka-connect-hdfs/quickstart-hdfs.properties`` are properly set to your configurations of
-Hadoop, e.g. ``hdfs.url`` points to the proper HDFS and using FQDN in the host. Then run the
-following command to start Kafka connect with the HDFS connector::
+Hadoop, e.g. ``hdfs.url`` points to the proper HDFS and using FQDN in the host. Then start connector by loading its
+configuration with the following command:
+
+.. sourcecode:: bash
+
+   $ confluent load hdfs-sink -d etc/kafka-connect-hdfs/quickstart-hdfs.properties
+   {
```

Comment:
This line starts the output of the `confluent load` command, right? I know people will think they have to run the whole command including what's in the braces. I suggest adding a normal text line after the command that says something like "This will output the connector configuration that is being started:"